### PR TITLE
Concourse update

### PR DESCRIPTION
**Pull request by halimacc**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
* Improve deployment routine for Concourse
* Upgrade versions
* Adjust project structure

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@halimacc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@halimacc__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: concourse-ci/README.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
 # Setup Concourse CI
 
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fconcourse-ci%2Fazuredeploy.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fhalimacc%2Fazure-quickstart-templates%2Fconcourse-update%2Fconcourse-ci%2Fazuredeploy.json" target="_blank">
```
Comment (bingosummer) [inline concourse-ci/README.md]: @halimacc 
Please remove your personal repo&branch info.

**Comment by bingosummer** [comment]
LGTM

**Comment by gatneil** [comment]
LGTM too :); merging

**Pull request merged**